src/bridge/bridge.c

```diff
@@ -682,12 +684,45 @@ main (int argc,
   signal (SIGABRT, cockpit_test_signal_backtrace);
   signal (SIGSEGV, cockpit_test_signal_backtrace);
 
-  /* In case we are run via sshd and we have journald, make sure all
-   * logging output ends up in the journal on *this* machine, not sent
-   * back to the client.
-   */
-  if (g_getenv ("SSH_CONNECTION") && !g_log_writer_is_journald (2) && ! isatty(2))
+  if (g_strv_contains ((const gchar **) argv, "--privileged"))
```

Comment:
Strictly speaking, we shouldn't use this corner of GLib either.....

Reply:
We already call other random little "harmless" bits of GLib for the ssh case, though.  This is probably fine, and better than the alternative.